catchpoint/dashboards/Catchpoint-Recent_Errors.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Ctachpoint  - Recent Errors",
+  "description": null,
```

Comment:
Same comment on this dashboard as the other one, running the sanitize dashboard script should clean that up for you